src/core/funcbloc.h

```diff
@@ -84,10 +84,12 @@ struct SCFB_FBInstanceData {
 struct SFBInterfaceSpec {
     TEventID mNumEIs; //!< Number of event inputs
     const CStringDictionary::TStringId *mEINames; //!< List of the event input names
+    const CStringDictionary::TStringId *mEITypeNames; //!< List of the event input types
```

Comment:
I suggest adding the additional fields at the end, so that existing code remains compatible.

Reply:
I don't think we should do that. Because the getEvent Type Methods will then undeteccted read from arbitrary memory.  We are working on a 3.0.0 we can now break some API.

Reply:
Why would they read from arbitrary memory? Any excess fields are implicitly initialized, see https://en.cppreference.com/w/cpp/language/aggregate_initialization#Implicitly_initialized_elements.